saleor/graphql/page/types.py

```diff
@@ -9,6 +9,13 @@
 
 
 class Page(CountableDjangoObjectType):
+    content = String(
+        description="Content for the page.",
+        deprecation_reason=(
+            "Will be removed in Saleor 3.0. Use the `contentJson` instead."
```

Comment:
This field won't be removed; we will change the type. Let's just say "Use the `contentJson` field instead".